### PR TITLE
Bhavani/vz 4694

### PR DESCRIPTION
**Pull request by bhavaniravichandran**
# Description

Please include a summary of the change and which issue is fixed.  If there are any dependencies, for example on a PR in another repository, please list those.

Fixes VZ-9999

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/lre/Jenkinsfile [hunk 1]
```diff
@@ -221,6 +229,33 @@ pipeline {
...
+                    steps {
+                        runGinkgo('examples/helidonconfig')
+                    }
+                }
+                stage('examples springboot') {
+                    steps {
+                        runGinkgo('examples/springboot')
+                    }
+                }
+		stage('examples socks') {
```
Comment (bobdonat) [inline ci/lre/Jenkinsfile:242]: nit. indentation is off

Comment (bhavaniravichandran) [reply ci/lre/Jenkinsfile]: fixed

**Review by stevshap — APPROVED** [review]

File: examples/bobs-books/bobs-books-comp.yaml [hunk 2]
```diff
@@ -28,7 +26,7 @@ spec:
             - name: BACKEND_PORT
               value: "8001"
             - name: BACKEND_HOSTNAME
-              value: bobs-bookstore-cluster-cluster-1.bobs-books.svc.cluster.local
+              value: bobs-bookstore-cluster-cluster-1
```
[the remark this answers is not in the document]
Comment (bhavaniravichandran) [reply examples/bobs-books/bobs-books-comp.yaml:29]: To keep the fully qualified hostname, we would have to templatize the yaml and we didn't want to use template yaml. The weblogic cluster service name is unique to the app and there will be no conflicts. So, just specifying the cluster svc name is sufficient

File: examples/todo-list/todo-list-components.yaml [hunk 3]
```diff
@@ -92,7 +88,7 @@ spec:
                   TestFrequencySeconds: 10
                 JDBCDriverParams:
                   # for MySQL, the last element in the URL is the database name, and must match the name inside the DB server
-                  URL: "jdbc:mysql://mysql.todo-list.svc.cluster.local:3306/tododb"
+                  URL: "jdbc:mysql://mysql:3306/tododb"
```
[the remark this answers is not in the document]
Comment (bhavaniravichandran) [reply examples/todo-list/todo-list-components.yaml:91]: same as above

File: tests/e2e/pkg/resources.go [hunk 4]
```diff
@@ -116,9 +116,95 @@ func createOrUpdateResourceFromBytes(data []byte, config *rest.Config) error {
 	// no return since you can't get here
 }
 
-func readNextResourceFromBytes(reader *utilyaml.YAMLReader, mapper *restmapper.DeferredDiscoveryRESTMapper, client dynamic.Interface, uns *unstructured.Unstructured) (*meta.RESTMapping, error) {
+// CreateOrUpdateResourceFromFileInGeneratedNamespace creates or updates a Kubernetes resources from a YAML test data file.
+// The test data file is found using the FindTestDataFile function.
```
Comment (markxnelson) [inline tests/e2e/pkg/resources.go:120]: I don't see it using FindTestDataFile?

**Comment by markxnelson** [comment]
hold this until the matching/compensating doc PR is also ready

**Review by markxnelson — APPROVED** [review]

**Pull request merged**